source/develop/release-management/features/network/networking-api-security-groups.html.md

```diff
@@ -0,0 +1,597 @@
+currently not supported in the ovirt-provider-ovn, and thus, will also
+have to be added to the API.
+
+*If* the port is attached to a network featuring a subnet, the
+corresponding **allow-dhcp** port group port list will also be updated
+with the new port.
+
+It will also need to update the list of ports for each of the port
+groups representing the security groups bound to the added ports.
+
+![add-port](../../../../images/features/network/security-groups/add_ports.png)
+
+## OVN ACLs implementing the deny-all *and* allow-dhcp security groups
+
+### drop-all-ip ACL
+
+To achieve the intended deny-all behavior, a port group will be created
+when the ovirt-provider-ovn starts.
+
+It will feature two ACLs, with the following data:
++ ingress rule
+  * priority: DROP_PRIORITY
+  * action: drop
+  * direction: from-lport
+  * match: inport == @<port_group> && ip
++ egress rule
+  * priority: DROP_PRIORITY
+  * action: drop
+  * direction: to-lport
+  * match: outport == @<port_group> && ip
+
+**NOTE:** these rules will be filtered out by the API, because they are simply
+not featured in the Networking API. This means that a user listing all security
+group rules **will not** see the rules corresponding to the ACLs defined above.
+
+### allow-dhcp
+
+To allow a VM to get an IP address from the subnet on top the network
+to which its port is attached, and assuming that port has port-security
+active, it is required to white-list the DHCP traffic.
+
+This will only be done *if* the VM's port is attached to a network
+**having** a defined subnet.
+
+It will feature two ACLs (one per each direction), with the following
+data:
++ ingress rule
+    * priority: ALLOW_PRIORITY
+    * action: allow
+    * direction: from-lport
+    * match: inport == @allow-dhcp<subnet_uuid> && ip4 && ip4.dst == {255.255.255.255, <subnet_cidr>} && udp && udp.src == 68 && udp.dst == 67
++ egress rule
+    * priority: ALLOW_PRIORITY
+    * action: allow
+    * direction: to-lport
+    * match: outport == @allow-dhcp<subnet_uuid> && ip4 && ip4.src == <subnet_cidr> && udp && udp.src == 67 && udp.dst == 68
+
+***Note:*** the '@' sign is a port group tag understood by OVN
+
+## ovirt-provider-ovn API update
+
+The following updates are required to the current API, listed by entities:
+
+* network
+  + add the **port_security_enabled** boolean attribute
+* port
+  + implement the **port_security_enabled** boolean attribute
+  + implement the **security_groups** array attribute
+* security groups
+  + implement this entity, as described [here](#encoding-the-security-group-information)
+* security group rules
+    + implement this entity, as described [here](#encoding-the-security-group-rule-information)
+
+## Networking API extensions implemented / not implemented
+
+Implemented:
+* [Resource timestamps](https://developer.openstack.org/api-ref/network/v2/#id367)
+* [Port security](https://developer.openstack.org/api-ref/network/v2/#id35)
+
+Not implemented:
```

Comment:
Do we want to leave it as is, or state if we intend on implementing / not supporting?

Reply:
Done